netket/models/equivariant.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union, Optional, Tuple, Any, Callable, Iterable
+
+import numpy as np
+
+import jax
+from jax import numpy as jnp
+from flax import linen as nn
+
+from netket.hilbert import AbstractHilbert
+from netket.graph import AbstractGraph, SymmGroup
+from netket.utils import HashableArray
+from netket.utils.types import PRNGKeyT, Shape, DType, Array, NNInitFunc
+
+
+from netket import nn as nknn
+from netket.nn.initializers import lecun_complex, zeros, variance_scaling
+
+
+class GCNN(nn.Module):
+    """Implements a group convolutional neural network with symmetry
+    averaging in the last layer as described in Roth et al. 2021."""
```

Comment:
Can you also write something more here?

Put a new line and then write a bit more, so the website documentation is actually helpful.
you can also write latex equations.

Reply:
Okay that is good to know I'll try to make these more helpful

Reply:
A good example of a complete documentation and using latex is in LocalLiouvillian.

In general everything you write before a newline is considered the short-summary used in short descriptions in the website, and then after the newline you can write everything else, generates the docpage.